src/redux/actions/projects/createProject.js

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment';
+import hash from 'object-hash';
+
+import {
+  PROJECTS_CREATE,
+} from '../../actionTypes/projects';
+import { projectTemplate } from '../../reducers/projects/initialState';
+
+const createProject = (
+  projectName,
+) => async (dispatch) => {
+  const createdAt = moment().local().format('DD MMM YYYY, HH:mm:ss [GMT]Z');
```

Comment:
This may not be the best idea from a UX point of view.

First of all, it is not easily localizable, many languages/locales don't use the type of format that is in here (`04 Mar 2021, 16:13:40 GMT+00:00`).

Second, it is in general not a good idea to tell people actual dates for when things happened, relative dates are easier to parse and think about. This also resolves the issue you had with timezones.

In general, the best practice would mean the following: for createdAt and lastModified, you can simply use the ISO string. This ensures cross-compatibility with different languages, and a universal way to store dates as text.

```
> const createdAt = moment().toISOString()
undefined
> createdAt
'2021-03-04T16:20:20.373Z'
```

Now, you need some way to display this on the UI. It is almost always a good idea when displaying dates on the UI to always display relative dates as the main string, but to also show the proper date if you hover over the text:

```
> moment(createdAt).fromNow()
'a minute ago'
```

For displaying dates in general in UI that are not relative (such as the full date on hover), it is a good idea to always use the localized version of the date so we will be able to later support different locales/regions. You can see [here](https://momentjs.com/docs/#/displaying/) that you can use `LLLL` as a format string that will automatically format the date according to the locale of the user. On my computer the default locale is `en-US` so I get (with no settings):

```
> moment(createdAt).format('LLLL')
'Thursday, March 4, 2021 4:20 PM'
```

But if we were to support say Polish, British English, or Indonesian in the future, we can do that too, and the same string adjusts to the way dates are usually written in that region:

```
> moment.locale('pl-PL')
'pl'
> moment(createdAt).format('LLLL')
'czwartek, 4 marca 2021 16:20'

> moment.locale('en-GB')
'en-gb'
> moment(createdAt).format('LLLL')
'Thursday, 4 March 2021 16:20'

> moment.locale('id-ID')
'id'
> moment(createdAt).format('LLLL')
'Kamis, 4 Maret 2021 pukul 16.20'
```

Reply:
Added in commit d78e4f3